app/src/main/java/com/eveningoutpost/dexdrip/Services/WixelReader.java

```diff
@@ -353,9 +355,17 @@ public void readData()
     	TransmitterData lastTransmitterData = TransmitterData.last();
     	if(lastTransmitterData != null) {
     	    LastReportedTime = lastTransmitterData.timestamp;
-    	}
+
```

Comment:
shouldn't this be DEXCOM_PERIOD+2000?
I believe that 298 seconds from last packet we will still not find anything.